.changeset/five-stingrays-search.md

```diff
@@ -0,0 +1,19 @@
+---
+'@graphql-hive/envelop': minor
+---
+
+Add support for providing a target for usage reporting with organization access tokens.
+This can either be a slug following the format "$organizationSlug/$projectSlug/$targetSlug" (e.g "the-guild/graphql-hive/staging")
+or an UUID (e.g. "a0f4c605-6541-4350-8cfe-b31f21a4bf80")
```

Comment:
I think at the moment we do not show this ID to the user, should we create a followup PR to add that somewhere in the dashboard?

Reply:
we already have it https://github.com/graphql-hive/console/issues/6466